templates/openstack/consulting.html

```diff
@@ -1,389 +1,501 @@
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">OpenStack design</h3>
+          <p class="p-matrix__desc">Together, we design the cloud according to your needs. We help you make right decisions regarding cloud architecture, hardware configuration, SDN and storage options to make sure that the cloud we build meets your requirements.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">OpenStack delivery</h3>
+          <p class="p-matrix__desc">We test your hardware, make sure the physical network is configured properly, build the cloud, validate it post-deployment, benchmark its performance and onboard your tenants.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Workloads migration</h3>
+          <p class="p-matrix__desc">We can optionally migrate your existing workloads to your new private cloud. We partner with market-leading companies that offer workloads migration with minimal downtime.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Repeatable artifacts</h3>
+          <p class="p-matrix__desc">At the end of the engagement we provide you with a detailed deployment guide that enables repeatable re-deployments of your private cloud and perform a formal handover process.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Day-2 automation</h3>
+          <p class="p-matrix__desc">With PCB, you are not left alone on day-2. Charmed OpenStack provides full automation around common day-2 maintenance tasks, including daily operations, scale-out capabilities, integration and OpenStack upgrades.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Optional training</h3>
+          <p class="p-matrix__desc">On request, we train your personnel to get them up to speed with your new private cloud. We provide comprehensive training courses that cover OpenStack architecture, implementation and operations.</p>
+        </div>
+      </li>
+    </ul>
+    <p class="u-sv3">
+      <a class="p-button--positive p-link--external" href="https://assets.ubuntu.com/v1/d4766546-Private-Cloud-Build_Datasheet.pdf">Download PCB datasheet</a>
+    </p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <blockquote class="p-pull-quote">
```

Comment:
```suggestion
    <blockquote class="p-pull-quote--large">
```
Is a little nicer on rows of there own